tests/unit/DissolutionFirm.spec.ts

```diff
@@ -138,5 +138,22 @@ for (const test of dissolutionFirmTestCases) {
       expect(errorMessages.at(0).text()).toBe('Cannot exceed 20 characters')
       expect(errorMessages.at(1).text()).toBe('Cannot exceed 20 characters')
     })
+
+    it('display correct date rules', async () => {
+      wrapper = mount(
+        DissolutionFirm,
+        { vuetify }
+      )
+      const rules = wrapper.vm.startDateRules
+      store.setCurrentJsDate(new Date('2022-06-14T12:00:00.000Z'))
+
+      expect(rules[0]('')).toBe('Dissolution date is required') // no date is selected
+      expect(rules[0]('October 16, 2023')).toBe(true) // date is selected
+      // A date before the registration date is selected (invalid)
+      expect(rules[1]('June 5, 2022')).toContain('Dissolution Date must be after June 6, 2022 and up to')
+      expect(rules[1]('June 5, 2022')).toContain('June 14, 2022')
+      // A valid date is selected (after registration date and not in the future)
+      expect(rules[1]('June 7, 2022')).toBe(true)
+    })
```

Comment:
Can you test the edge cases, too, please?

Example:  "Jun 5 0:00:00 >= Jun 5" and "Jun 6 23:59:59 <= Jun 6"?

Reply:
On it!

Reply:
What about the other edge? (future date)

Reply:
Done haha. Was exactly thinking that. Sorry about that Sev.

Reply:
Sev, the tests are running locally but failing here. I can't seem to figure out why. Could it be related to timezones?

Reply:
Never mind, I fixed it 😁 My suspicions were correct, it was timezone issue. Should be fine now!

Reply:
Timezone issues 🤯